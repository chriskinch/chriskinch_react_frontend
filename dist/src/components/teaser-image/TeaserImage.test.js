'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _TeaserImage = require('./TeaserImage');

var _TeaserImage2 = _interopRequireDefault(_TeaserImage);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('Snapshot renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_TeaserImage2.default, { path: 'http://chriskinch.netlify.com', alt: 'Teaser Image', title: 'Grey pixel' })).toJSON();
  expect(tree).toMatchSnapshot();
});

//# sourceMappingURL=TeaserImage.test.js.map