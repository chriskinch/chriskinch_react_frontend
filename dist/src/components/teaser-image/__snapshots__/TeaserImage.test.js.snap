// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders correctly 1`] = `
<div
  className="field field--name-field-teaser-image field--type-image field--label-hidden field__item"
>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    <picture>
      <source
        media="(max-width: 600px)"
        srcSet="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
      />
      <img
        alt="Teaser Image"
        height={500}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
        title="Grey pixel"
        typeof="foaf:Image"
        width={500}
      />
    </picture>
  </a>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="field field--name-field-teaser-image field--type-image field--label-hidden field__item"
>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    <picture>
      <source
        media="(max-width: 600px)"
        srcSet="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
      />
      <img
        alt="Teaser Image"
        height={500}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
        title="Grey pixel"
        typeof="foaf:Image"
        width={500}
      />
    </picture>
  </a>
</div>
`;
